README.md

```diff
@@ -5,6 +5,19 @@
 
 An example mod for Minecraft 1.7.10 with Forge focussed on a stable, updatable setup.
 
+### Motivation
+
+We had our fair share in struggles with build scripts for Minecraft Forge. There are quite a few pitfalls from non-obvious error messages. This Example Project provides you a build system you can adapt to over 90% of Minecraft Forge mods and can easily be updated if need be.
+
+### Getting started
+
+1. Fork this repository (you can keep the fork to simplify merging any build script updates in the future)
+2. Clone repository
+3. Replace placeholders (e.g. modid, author(s) name, etc.)
```

Comment:
I'd rather say something like "Work through gradle.properties". Something to make sure that file is read

Reply:
done

Reply:
also for this case I've added a point which says to checkout other sections, so I assume they will be interested enough to checkout all the files you've listed

Reply:
Let's hope for it!